config/default/core.entity_view_display.paragraph.civictheme_message.default.yml

```
@@ -0,0 +1,68 @@
+targetEntityType: paragraph
+bundle: civictheme_message
+mode: default
+content:
+  field_c_p_background:
+    type: boolean
+    label: above
+    settings:
+      format: default
+      format_custom_false: ''
+      format_custom_true: ''
+    third_party_settings: {  }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)